api/remove_all_policies.md

```diff
@@ -0,0 +1,54 @@
+---
+api_name: remove_all_policies()
+excerpt: Remove all policies from a continuous aggregate
+topics: [continuous aggregates, jobs, compression, data retention]
+keywords: [continuous aggregates, policies, remove, compress, data retention]
+api:
+  license: community
+  type: function
+  experimental: true
+---
+
+import Experimental from "versionContent/_partials/_experimental.mdx";
+
+<!-- markdownlint-disable-next-line line-length -->
+# remove_all_policies() <tag type="community" content="Community" /><tag type="experimental" content="Experimental" />
+
+Remove all policies from a continuous aggregate. The removed compression and
+retention policies apply to the continuous aggregate, _not_ to the original
+hypertable.
+
+```sql
+timescaledb_experimental.remove_all_policies(
+     relation REGCLASS,
+     if_exists BOOL = false
+) RETURNS BOOL
+```
+
+<Experimental />
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`relation`|`REGCLASS`|The continuous aggregate to remove all policies from|
+
+## Optional arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`if_exists`|`BOOL`|When true, prints a warning instead of erroring if no policies exist. Defaults to false.|
```

Comment:
@RafiaSabih Is it "if _no_ policies exist" or does it also error if any one of the policies doesn't exist?